src/site/components/banner_alerts.drupal.liquid

```diff
@@ -0,0 +1,76 @@
+{% for entity in alert.entities %}
+{% assign dismissStatus = entity.fieldAlertInheritanceSubpages %}
+{% assign alertType = entity.fieldAlertType %}
+
+{% if alertType = "information" %}
+{% assign alertType = "info" %}
+{% endif %}
+
+{% assign region = path | split: "/" | first | prepend: "/" %}
+{% assign lastArg = path | split: "/" | last | prepend: "/"%}
+
+{% assign outputStatus = false %}
+{% assign emailUpdates = "https://www.va.gov/" %}
+{% assign statusUrl = "https://www.va.gov/" %}
+
+{% for vamc in entity.fieldBannerAlertVamcs %}
+
+{% if region == vamc.entity.fieldOffice.entity.entityUrl.path %}
+
+{% assign outputStatus = true %}
+
+{% if dismissStatus == true and lastArg != region and lastArg != "/operating-status" %}
+{% assign outputStatus = false %}
+{% endif %}
+{% assign emailUpdates = vamc.entity.fieldOffice.entity.fieldLinkFacilityEmergList.uri %}
+{% assign statusUrl = vamc.entity.entityUrl.path %}
+{% endif %}
+{% endfor %}
+
+{% if outputStatus == true %}
+<div aria-labelledby="usa-alert-heading-{{ entity.entityId }}"
+  class="usa-alert-full-width dismissable-option-header usa-alert-full-width-{{ entity.fieldAlertType }}"
+  id="usa-alert-full-width-{{ entity.entityId }}" role="region">
+  <div class="usa-alert usa-alert-{{ alertType }}" role="alert">
+    {% if entity.fieldAlertDismissable == true %}
+    <button id="usa-alert-dismiss-{{ entity.entityId }}"
+      class="va-alert-close usa-alert-dismiss"
+      data-parentwrap="usa-alert-full-width-{{ entity.entityId }}"
+      data-frequency="once" aria-label="Close notification">
+      <i aria-label="Close icon" class="fas fa-times-circle"></i>
+    </button>
+    {% endif %}
+    <div class="usa-alert-body">
+      <h3 class="usa-alert-heading">
+        {{ entity.title }}
+      </h3>
+      <div class="additional-info-content usa-alert-text">
+        {{ entity.fieldBody.processed }}
+
+        {% if entity.fieldAlertOperatingStatusCta == true %}
+        <p>
+          <a href="{{statusUrl}}">Get
```

Comment:
Does this work, or does it need spaces around the variable name? (`{{ statusUrl }}`)

Reply:
It does, but Updated @cvalarida :+1: